python/cucim/MANIFEST.in

```diff
@@ -1,24 +1,13 @@
 # https://packaging.python.org/guides/using-manifest-in/
-graft docs
-graft src
-graft ci
-graft tests
-
-include .bumpversion.cfg
-include .coveragerc
-include .cookiecutterrc
 include .editorconfig
 
-include VERSION
-include AUTHORS.md
+include src/cucim/VERSION
 include CHANGELOG.md
 include CONTRIBUTING.md
-include ../../LICENSE
-include ../../LICENSE-3rdparty.md
+include LICENSE
+include LICENSE-3rdparty.md
 include README.md
```

Comment:
Should we convert `README.md` to a symlink to the top-level `README.md` as well?

Reply:
This is now a symlink